pytorch_toolkit/object_detection/tests/export_tests_custom_detection.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (C) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import os
+
+import yaml
+from ote.tests.utils import collect_ap, run_through_shell
+
+from common.test_case import create_object_detection_export_test_case
+
+
+kwargs = dict(
+    problem_name='custom-object-detection',
+    ann_file=os.path.dirname(__file__) + '/../../../data/airport/annotation_example_train.json',
+    img_root=os.path.dirname(__file__) + '/../../../data/airport/train',
+    alt_ssd_export=True
+)
+
+
+def create_custom_object_detection_export_test_case(model_name):
+    class ExportTestCase(create_object_detection_export_test_case(model_name=model_name, **kwargs)):
+
+        classes = 'vehicle,person,non-vehicle'
+
+        def do_evaluation(self, export_dir):
+            metrics_path = os.path.join(export_dir, "metrics.yaml")
+            run_through_shell(
+                f'cd {os.path.dirname(self.template_file)};'
+                f'python eval.py'
+                f' --test-ann-files {self.ann_file}'
+                f' --test-data-roots {self.img_root}'
+                f' --load-weights {os.path.join(export_dir, "model.bin")}'
+                f' --save-metrics-to {metrics_path}'
+                f' --classes {self.classes}'
+            )
```

Comment:
I would add `TODO` here that more tricky approach should be invented to avoid copy of code in methods `do_evaluate` etc.
(At the moment I do not have a decision, but most probably it exists)

Reply:
I'm afraid this TODO will stay here forever. Don't see here significant reason to refactor this now. If there are more examples and more duplication then yes.

Reply:
I think it is better to have this TODO. In this case the probability that we recall that refactoring is required is higher.

Reply:
Ilya, I accept this PR, since (as far as I can see) it is generally ok.

But, nevertheless, I recommend to add TODO here -- I really think it may help in the future when we will decide what should be refactored
(I hope we will do it)